gifski.h

```diff
@@ -53,6 +53,10 @@ typedef struct GifskiSettings {
    * Lower quality, but faster encode.
    */
   bool fast;
+  /**
+   * 0+, but useful range is 1-10. Recommended to set to 5. If loop_count is > 0 once should be set to true
```

Comment:
> If loop_count is > 0 once should be set to true

This contradicts the actual code where it's set automatically:

https://github.com/ImageOptim/gifski/pull/150/files#diff-2c5e283f768834edb2a16a141ffd34d1a9f716f87f878198ac40707a1c01b29bR159

Reply:
Add a useful description of its purpose. And end it with a `.`